tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/util/Wait.java

```diff
@@ -41,13 +41,13 @@ public static boolean waitFor(final Condition condition, final long duration) th
 
    public static boolean waitFor(final Condition condition,
                                  final long duration,
-                                 final int sleepMillis) throws Exception {
```

Comment:
This doesn't seem related?


It's a minor change and I don't mind.. just checking if I'm missing anything.